sdk/modelparser/azure-digitaltwins-parser/src/main/java/com/azure/digitaltwins/parser/implementation/codegen/JavaEnum.java

```diff
@@ -14,10 +14,10 @@
  * Generator for java enum
  */
 public class JavaEnum extends JavaDeclaration implements JavaFile {
-    private static ClientLogger logger = new ClientLogger(JavaEnum.class);
-    private List<EnumVal> enumValues;
-    private boolean isSorted;
-    private String typeName;
+    private static final ClientLogger LOGGER = new ClientLogger(JavaEnum.class);
```

Comment:
why change to upper case?

Reply:
I made it static final and static final fields need to be all upercase.